sysutils/pagerduty-notifier/Portfile

```
@@ -0,0 +1,78 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/trustpilot/pagerduty-notifier 20200817-0652
+
+description         PagerDuty native MacOS notifications
+
+long_description    PagerDuty Notifier polls the PagerDuty api for new \
+                    incidents and sends native Mac OS notifications. You can \
+                    control the notification style as you normally would in \
+                    System Preference
+
+categories          sysutils
+license             MIT
+
+maintainers         {gmail.com:herby.gillot @herbygillot} \
+                    openmaintainer
+
+installs_libs       no
+
+build.cmd           make
+build.target        build
+build.env-append    GO111MODULE=off \
+                    GOPROXY=off
+destroot {
+    file copy ${worksrcpath}/dist/PagerdutyNotifier.app \
+        ${destroot}${applications_dir}/
+}
+
+notes "PagerdutyNotifier installed to Applications/MacPorts"
```

We would not normally add notes telling the user where things are installed. The user is expected to know to use `port contents` to find out. If you feel it essential to print this note, then at least print the right location. (`applications_dir` can be configured to be something other than /Applications/MacPorts.)
```suggestion
notes "PagerdutyNotifier is installed to ${applications_dir}"
```

---

This is an app bundle which needs to be started through Finder, and this isn't clearly called out even in the original site.  But even so, I've opted to remove the notes.